apps/dapp/src/components/Pages/Core/NewUI/Home.tsx

```diff
@@ -92,33 +93,33 @@ const Home = ({ tlc }: { tlc?: boolean }) => {
   useEffect(() => {
     const fetchMetrics = async () => {
       const { data: treasuryData } = await fetchGenericSubgraph<any>(
-        'https://api.thegraph.com/subgraphs/name/medariox/temple-metrics',
+        env.subgraph.protocolMetrics,
         `{
           metrics {
             treasuryValueUSD
           }
         }`
       );
       const { data: arbitrumTreasuryData } = await fetchGenericSubgraph<any>(
-        'https://api.thegraph.com/subgraphs/name/medariox/temple-metrics-arbitrum',
+        env.subgraph.protocolMetricsArbitrum,
         `{
           metrics {
             treasuryValueUSD
           }
         }`
       );
       const { data: ramosData } = await fetchGenericSubgraph<any>(
-        'https://api.thegraph.com/subgraphs/name/templedao/templedao-ramos',
+        env.subgraph.protocolMetrics,
         `{
           metrics {
-            treasuryPriceIndexUSD
-            templePriceUSD
+            treasuryPriceIndex
+            templePrice
           }
         }`
       );
       setMetrics({
-        price: parseFloat(ramosData.metrics[0].templePriceUSD),
-        tpi: parseFloat(ramosData.metrics[0].treasuryPriceIndexUSD),
+        price: parseFloat(ramosData.metrics[0].templePrice),
```

Comment:
Please use the `ramos` subgraph for the price, see [example query](https://api.thegraph.com/subgraphs/name/templedao/templedao-ramos/graphql?query=query+MyQuery+%7B%0A++metrics+%7B%0A++++spotPrice%0A++%7D%0A%7D).

Reply:
Done https://github.com/TempleDAO/temple/pull/919/commits/4d016313dc0708a24fd0946725faa519ad1b49b6